pkg/virt-controller/watch/application.go

```diff
@@ -0,0 +1,129 @@
+package watch
+
+import (
+	"flag"
+	golog "log"
+	"net/http"
+	"strconv"
+
+	"github.com/emicklei/go-restful"
+	clientrest "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+
+	"k8s.io/client-go/util/workqueue"
+
+	"k8s.io/client-go/kubernetes"
+
+	kubeinformers "kubevirt.io/kubevirt/pkg/informers"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/logging"
+	"kubevirt.io/kubevirt/pkg/virt-controller/rest"
+	"kubevirt.io/kubevirt/pkg/virt-controller/services"
+)
+
+var (
+	flags           Flags = Flags{}
```

Comment:
In general, yes.  For virt-controller, however, there is only ever one of these.  Making them globals means they can be shared between main() and the unit tests.

I'm willing to go further on this if I can get a pre-approval on the approach.